content/en/users/cloud-compute/gpgpu/_index.md

```diff
@@ -18,28 +18,41 @@ The table below summarises the available options:
-| IISAS-Nebula   | up to 2 NVIDIA Tesla K20m              | resource_tpl#large_gpu                                                                                                                                                                     | acc-comp.egi.eu, biomed                                    | Sponsored access for limited testing, conditions to be negotiated for long-term usage |
+| IISAS-FedCloud | up to 2 NVIDIA Tesla K20m              | g1.c08r31-K20m, g1.c16r62-2xK20m                                                                                                                                                           | acc-comp.egi.eu, eosc-synergy.eu, enmr.eu, training.egi.eu | Sponsored access for limited testing, conditions to be negotiated for long-term usage |
 | IFCA-LCG2      | up to 2 NVIDIA T4, up to 2 NVIDIA V100 |                                                                                                                                                                                            |                                                            | Pay-per-use                                                                           |
 | CESNET-MCC     | up to 2 NVIDIA T4                      | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.38core-372ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.38core-372ram-nvidia-2080-glados | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com       | Sponsored, conditions to be negotiated                                                |
 
 <!-- markdownlint-enable line-length -->
 
 ## Access to GPU resources on EGI Cloud
 
-GPUs sites can be accessed by different ways: via site-specific dashboards and
-endpoints or via common federated-cloud services like common Horizon dashboard,
-[VMOps dashboard](../vmops), or [Infrastructure manager](../im).
+GPUs sites can be accessed in different ways: via site-specific dashboards and
+endpoints or via common federated-cloud services like the OpenStack Horizon
+dashboards, [VMOps dashboard](../vmops), or [Infrastructure manager](../im).
+
+It is also possible to use the
+[fedcloudclient](https://fedcloudclient.fedcloud.eu/) for CLI access. Below is
+an example on how to use the `fedcloud` command to show the GPU properties of
+the available flavors:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+fedcloud openstack flavor list --long --site IISAS-FedCloud --vo acc-comp.egi.eu --json-output | \
+jq -r '.[].Result | map(select(.Properties."Accelerator:Type" == "GPU")) | .'
+```
+
+<!-- markdownlint-enable line-length -->
```

Comment:
Instead of disabling the line length check here, it would be easier/cleaner to just add some more line returns with backslash in the shell excerpt.
The line following the return should also be indented.